j2/rules/webiny_dsar.yaml

```diff
@@ -0,0 +1,105 @@
+# DO NOT UPDATE THIS FILE MANUALLY; ANY CHANGES WILL BE OVERRIDDEN BY THE J2 DIR TEMPLATES
+updatereq:
+  vertices:
+    
+    - label: Person_Natural
+      condition: ${requisicaoTitularesCpf?.replaceAll('[^0-9]', '')?.trim()?.length() == 11}
+      props:
+        - name: Person_Natural_Customer_ID
+          val: ${requisicaoTitularesCpf?.replaceAll('[^0-9]', '')?.trim()}
+          mandatoryInSearch: True
+    
+    - label: Object_Data_Source
+      props:
+        - name: Object_Data_Source_Name
+          val: WEBINY/REQUISIÇÕES-DOS-TITULARES
+          mandatoryInSearch: True
+    
+    - label: Event_Subject_Access_Request
+      props:
+        - name: Event_Subject_Access_Request_Form_Id
+          val: ${id}
+          mandatoryInSearch: True
+        - name: Event_Subject_Access_Request_Id
+          val: ${requisicaoTitularesEmail}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Event_Subject_Access_Request_Metadata_Update_Date
+          val: ${savedOn}
+          mandatoryInSearch: False
+          type: "java.util.Date"
+          excludeFromSearch: True
+        - name: Event_Subject_Access_Request_Metadata_Create_Date
+          val: ${createdOn}
+          mandatoryInSearch: False
+          type: "java.util.Date"
+          excludeFromSearch: True
+        - name: Event_Subject_Access_Request_Request_Type
+          val: ${def arr = []; for(item in requisicaoTitularesQualidadeObj){arr.add(item.key?.toString()?.replace('requisicaoTitularesQualidadeObj', '')+':'+item.value?.toString()?.capitalize())};return arr}
```

Comment:
Hmmm, I'm not convinced this is correct, as the object value will be true/false, and we need Read/write/update/delete.  Let's create a function to parse this.